Rachis/Rachis/RaftEngine.cs

```diff
@@ -382,6 +382,9 @@ public Task AddToClusterAsync(NodeConnectionInfo node)
 
         public Task UpdateNodeAsync(NodeConnectionInfo node)
         {
+            if (State == RaftEngineState.Leader && node.IsNoneVoter)
```

Comment:
`IsNoneVoter` is a strange property name, should probably be `IsNotVoting` or something like that